.github/workflows/update-command-manual.yml

```diff
@@ -0,0 +1,48 @@
+name: Update Ockam Command Manual
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-manual:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Ockam repository
+        uses: actions/checkout@v3
+
+      - name: Set up Git
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@github.com'
+
+      - name: Clone Ockam Documentation repository
+        run: |
+          git clone https://github.com/build-trust/ockam-documentation.git
+          cd ockam-documentation
+          git checkout command
+
+      - name: Generate Command Manual
+        run: |
+          cd ockam-documentation/manual/template/mdbook
+          chmod +x build.sh
+          ./build.sh
+
+      - name: Commit and Push Changes
+        run: |
+          cd ockam-documentation
+          git checkout -b update-command-manual
```

Comment:
Should the branch name contain the release tag as well to prevent collisions?

Reply:
Yeah,
We can do that to make things more clear and to the point.